### ISSUE TITLE
test_heka_sandbox: pb_len expected: 306 received: 310

### ISSUE THREAD

**Issue by sathieu** [mozilla-services/lua_sandbox#161]
Since 1.2, ctest is failing on my machine:
```
test 11
      Start 11: test_heka_sandbox

11: Test command: /home/mathieu/src/github.com/mozilla-services/lua_sandbox/release/src/heka/test/test_heka_sandbox
11: Test timeout computed to be: 1500
11: tests and results are mis-matched
11: tests and results are mis-matched
11: tests and results are mis-matched
11: pb_len expected: 306 received: 310
11: \x0a\x10E\x10<>,\xb8D\xc6\xa6s\x03\xd2Ka\xfe\xa4\x10\x83\x89\xc6\xbb\xee\x90\xac\xbe\x14\x1a\x0einject_payload2\xe6\x01abcdefghijklmnop\x1e\x03\x08\xd0\x01\x1f\x0a\x10abcdefghijklmnop\x10\x80\x94\xeb\xdc\x03\x1a\x04type"\x06logger(\x092\x07payload:\x0benv_versionJ\x08hostnameR\x13\x0a\x06number\x10\x039\x00\x00\x00\x00\x00\x00\xf0?R,\x0a\x07numbers\x10\x03\x1a\x05count:\x18\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x00\x08@R\x0e\x0a\x05bools\x10\x04B\x03\x01\x00\x00R\x0a\x0a\x04bool\x10\x04@\x01R\x10\x0a\x06string"\x06stringR\x15\x0a\x07strings"\x02s1"\x02s2"\x02s3J\x0aultrathieuR\x13\x0a\x0cpayload_type"\x03txt
11: line: 725 (0 == rv) expected: 0 received: 1 process_message() lua/read_message_zc.lua:61: inject_message() failed: rejected by the callback
11: Tests run: 18
11/14 Test #11: test_heka_sandbox ..................***Failed    0.17 sec
```

**Comment by sathieu** [comment]
Is it 9dde916d719111991cd66896b70f1d1bd25ce70e?

**Comment by trink** [comment]
This was actually fixed Oct 6... it would have been nice if I push it to the repo from my centos box.
Addressed in: 70fda4a7d12fc2611218da7aa2d0f754f53735db
